src/docs/guide/usage/formatter/migrate-from-prettier.md

```diff
@@ -0,0 +1,279 @@
+# Migrating from Prettier to oxfmt
+
+If you currently use Prettier as your code formatter, you can follow this guide to migrate to oxfmt.
+
+Note that oxfmt is in alpha, and may not be suitable for production use in complex setups.
```

Comment:
Should maybe also add a note here at the top about the future addition of an automatic migration command, with a link to https://github.com/oxc-project/oxc/issues/15849